README.md

```diff
@@ -113,21 +114,14 @@ $ simple-ec2 launch
 
 +--------------------------------------+--------------------------+
 | Region                               | us-east-1                |
-+--------------------------------------+--------------------------+
 | VPC                                  | vpc-example              |
-+--------------------------------------+--------------------------+
-| Subnet                               | subnet-example 	  |
-+--------------------------------------+--------------------------+
+| Subnet                               | subnet-example 	        |
```

Comment:
`|` 👀

Reply:
Fixed in 60467e644a682e27cca61b8fee9f3eb503c338c2